contract/src/lib.rs

```diff
@@ -412,6 +426,45 @@ impl Contract {
             false
         }
     }
+
+    /// Public call to run GC on a mainchain.
+    /// batch_size is how many block headers should be removed in the execution
+    ///
+    /// # Panics
+    /// If initial blockheader or tip blockheader are not in a header pool
+    pub fn run_mainchain_gc(&mut self, batch_size: u64) {
+        let initial_blockheader = self
+            .headers_pool
+            .get(&self.mainchain_initial_blockhash)
+            .expect("initial blockheader must be in a header pool");
```

Comment:
Please replace it with  `.unwrap_or_else(|| env::panic_str("your error"))`

see this https://docs.near.org/build/smart-contracts/anatomy/reduce-size

Reply:
Changed `expect()` to `panic_str` in https://github.com/aurora-is-near/btc-light-client-contract/pull/9/commits/2c765163713994bb6cdd22daf20eba487a1b2c68